bin/rails

```diff
@@ -1,9 +1,4 @@
 #!/usr/bin/env ruby
-begin
-  load File.expand_path('../spring', __FILE__)
-rescue LoadError => e
-  raise unless e.message.include?('spring')
-end
```

Comment:
why removing spring?

Reply:
It was when we were having issues with double clicks.  I'll put it back.

Reply:
the double click issue is fixed now 🙂